distribution/docs/src/main/jdocs/content/_security-services/external-ws-s-sts-support-contents.adoc

```diff
@@ -0,0 +1,16 @@
+:type: securitySts
+:status: published
+:title: External/WS-S STS Support
+:link: _external_ws_s_sts_support
+:order: 02
```

Comment:
Probably. They aren't used for anything by this template, but I think they are useful for future navigating.